Forta-Bots-Challenge-2/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "Uniswap V3 Swap Detection Bot",
```

```suggestion
  "name": "uniswap-v3-swap-detection-bot",
```
I should've specified better, but with this particular property of the `package.json`, the `"name"` has to be formatted a certain way. lower-case and kebab-case. Great that you updated it though. 👍